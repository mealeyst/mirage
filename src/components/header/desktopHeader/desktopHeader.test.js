import React from 'react'
import 'jest-styled-components'

import { DesktopHeader } from 'SRC'

const { shallowWithTheme } = global

describe('(Styled component) DesktopHeader', () => {
  const createDesktopHeader = (props) => {
    return shallowWithTheme(<DesktopHeader {...props} />)
  }

  test('matching the snapshot', () => {
    expect(createDesktopHeader())
    .toMatchSnapshot()
  })
})
