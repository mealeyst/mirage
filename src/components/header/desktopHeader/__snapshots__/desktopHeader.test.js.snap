// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Styled component) DesktopHeader matching the snapshot 1`] = `
.c0 {
  background-color: #FFFFFF;
  min-height: 6rem;
  width: 100%;
  position: static;
  box-shadow: 0 2px 1px 0 rgba(0,0,0,.16);
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  top: 0;
  left: 0;
}

<header
  className="c0"
/>
`;
